src/schemas/json/github-issue-forms.json

```diff
@@ -1216,29 +1215,34 @@
     }
   },
   "type": "object",
+  "required": [
+    "name",
+    "description",
+    "body"
+  ],
   "properties": {
     "name": {
-      "description": "A name for the issue form template. Must be unique from all other templates, including Markdown templates.",
+      "description": "An issue template name\nhttps://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms#top-level-syntax",
```

Comment:
I'm not sure it adds to the user experience that you now have to click a link to see the details of the property. Any particular reason to not have both the details AND the link to more information?

Reply:
Fixed ;)